internal/pkg/utils/testhelper/env.go

```diff
@@ -35,12 +35,12 @@ func MustReplaceEnvsStringWithSeparator(str string, provider EnvProvider, envSep
 }
 
 // ReplaceEnvsStringWithSeparator replaces ENVs in given string with chosen separator.
-func ReplaceEnvsStringWithSeparator(str string, provider testtemplateinputs.EnvProvider, envSeparator string) (string, error) {
+func ReplaceEnvsStringWithSeparator(str string, provider EnvProvider, envSeparator string) (string, error) {
```

Comment:
Tu sa nejak omylom dostal konkretny typ, namiesto interface.

Ale pouzitelne to ma byt vsade, nie iba pre `templateinputs`.